velox/exec/Aggregate.h

```diff
@@ -417,6 +417,31 @@ using AggregateFunctionFactory = std::function<std::unique_ptr<Aggregate>(
     const TypePtr& resultType,
     const core::QueryConfig& config)>;
 
+struct AggregateFunctionMetadata {
+  /// Boolean indicating whether this aggregate function is order sensitive,
```

Comment:
This comment is very long, but it doesn't explain what orde-sensitive means to readers who do not already know it. I suggest to shorten this comment to

```
/// True if results of the aggregation depend on the order of inputs. For example, array_agg is order sensitive while count is not.
```

Reply:
Done